db/seeds.rb

```diff
@@ -1,8 +1,11 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-#
+AdminUser.create(email: 'admin@example.com', password: 'password')
+
+# Old Events
+Event.create(description: 'This is an old event. It happened last week. We had a greate time.', starts_at: 1.week.ago, url: 'http://www.example.com')
+Event.create(description: 'This is an older event. It happened two weeks. It was a blast!', starts_at: 2.weeks.ago, url: 'http://www.example.com')
+Event.create(description: 'This is an ancient event. It happened last year.', starts_at: 1.year.ago, url: 'http://www.example.com')
+
+# Future Events
+Event.create(description: 'This month we will talk about Ruby on Rails.', starts_at: 1.day.from_now, url: 'http://www.example.com')
+Event.create(description: 'Next month there is a great Javascript presentation.', starts_at: 1.month.from_now, url: 'http://www.example.com')
+Event.create(description: 'Eventaully there will be a talk on PHP.', starts_at: 1.year.from_now, url: 'http://www.example.com')
```

Comment:
Line is too long. [127/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.